src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsFromStaticMethodRefInDocsOnly.java

```diff
@@ -0,0 +1,22 @@
+/*
+UnusedImports
+processJavadoc = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.util.Collections;
+import static java.util.Collections.emptyEnumeration; // violation 'Unused import - java.util.Collections.emptyEnumeration.'
+
+public class InputUnusedImportsFromStaticMethodRefInDocsOnly {
+
+    /**
+     * Use {@link Collections::emptyEnumeration} in this javadoc.
+     * @see Collections::emptyEnumeration
```

Comment:
it might be discussed already.

Why this is violation ? 
https://checkstyle.org/config_imports.html#UnusedImports states it counted like usage.

Reply:
This is not a valid link

Reply:
> Do we know what is a valid link in javadoc?

As far as this check is concerned, we build identifiers from Javadoc comments, and check them just like identifiers in code.

Reply:
I became confused.
Please educate me how write good comment to be  considered by Check as usage.

Let's make Input and test on it to be explicit. If it is subtle nuance, we need to mention this in doc  to avoid bunch of issues from confused users